pkg/chunked/bloom_filter.go

```diff
@@ -0,0 +1,81 @@
+package chunked
+
+import (
+	"encoding/binary"
+	"hash/crc32"
+	"io"
+)
+
+type bloomFilter struct {
+	bitArray []uint64
+	k        uint32
+}
+
+func newBloomFilter(size int, k uint32) *bloomFilter {
+	numElements := (size + 63) / 64
+	return &bloomFilter{
+		bitArray: make([]uint64, numElements),
+		k:        k,
+	}
+}
+
+func newBloomFilterFromArray(bitArray []uint64, k uint32) *bloomFilter {
+	return &bloomFilter{
+		bitArray: bitArray,
+		k:        k,
+	}
+}
+
+func (bf *bloomFilter) hashFn(item []byte, seed uint32) (uint64, uint64) {
+	mod := uint32(len(bf.bitArray) * 64)
+	seedSplit := seed % mod
```

Comment:
Shouldn’t this be
```suggestion
	seedSplit := seed % len(item)
```
?

---

(I guess we don’t need to _actually_ care about zero-length items.)